api_tests/package.json

```diff
@@ -9,6 +9,7 @@
         "dry": "jest --config jest.config-dry.js --maxWorkers=4",
         "e2e": "yarn jest --config jest.config-e2e.js --runInBand --forceExit --bail",
         "test": "yarn dry && yarn e2e",
+        "ci": "tsc && yarn dry && yarn e2e",
```

Comment:
This extra yarn target is needed because the for `test` the target `pretest` will be triggered. This would build cnd again. We don't want that because we reuse the binary from the ubuntu build.